From: Nigel Cunningham <nigel@tuxonice.net>
Date: Fri, 25 Jan 2008 20:51:56 +0000 (+1100)
Subject: [TuxOnIce] Fix regression in starting userui (wrong umh_wait arg).
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fnigelc%2Ftuxonice-head.git;a=commitdiff_plain;h=c43ab8880c53f2acb55be4614ce97b8c9d3664c2

[TuxOnIce] Fix regression in starting userui (wrong umh_wait arg).

Signed-off-by: Nigel Cunningham <nigel@tuxonice.net>
---

diff --git a/kernel/power/tuxonice_netlink.c b/kernel/power/tuxonice_netlink.c
index 80f6c38..f459a7f 100644
--- a/kernel/power/tuxonice_netlink.c
+++ b/kernel/power/tuxonice_netlink.c
@@ -302,7 +302,7 @@ int toi_netlink_setup(struct user_helper_data *uhd)
 	}
 
 	if (toi_launch_userspace_program(uhd->program, uhd->netlink_id,
-				UMH_NO_WAIT) < 0) {
+				UMH_WAIT_EXEC) < 0) {
 		printk(KERN_INFO "Launch userspace program failed.\n");
 		toi_netlink_close_complete(uhd);
 		return 1;
